actions/download_samplesheet_clarity.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: "download_samplesheet_clarity"
+runner_type: "python-script"
```

Comment:
I think this got changed in 2.0 (?):
https://docs.stackstorm.com/2.0/reference/runners.html?highlight=runners#python-runner-python-script
also the action executes on the vagrant machine